server/04-server-base/requirements.txt

```diff
@@ -2,5 +2,6 @@ numpy
 pandas
 numba
 deephaven-plugin>=0.3.0
+jedi=0.18.1
```

Comment:
I'd prefer to not be explicit here. On the consumption side (deephaven-core), the version will be pinned as a guard against implicit updates. Do/will we have relevant tests against jedi on the deephaven-core side?

Reply:
where would I add tests that need to run in a fully setup venv?

Reply:
and... jedi is somewhat picky about versions, and does release breaking changes on minor version updates as it is `0.X`.  It's failing b/c I needed `==` not `=`.

Up to you if I just remove it, but I'd at least like to do `>= 0.18.1`...

Reply:
Okay - given the brittleness, let's go ahead and set `jedi==0.18.1`. On the deephaven-core side, @jmao-denver can help get some tests setup over there. Once we have some tests, we can relax this version restriction on the base image, pick up natural version bumps to jedi, and if the unit tests break, can either fix the breakages, or re-pin the version.

Reply:
ok, fixed.

I tried out 0.18.2 and it was fine, so pinning to that.